### ISSUE TITLE
maatwebsite/excel[3.1.0, ..., 3.1.25] require php ^7.0 -> your php version (8.2.13) does not satisfy that requirement.

### ISSUE THREAD

**Issue by DrewNeon** [pxlrbt/filament-excel#160]
When installing this package, the following problem occurs. Does it due to the dependent package? Is it solvable by downgrading PHP only?

`maatwebsite/excel[3.1.0, ..., 3.1.25] require php ^7.0 -> your php version (8.2.13) does not satisfy that requirement.`

**Comment by pxlrbt** [comment]
No, it definitely supports PHP8.2 in a higher version (see the repo: https://github.com/SpartnerNL/Laravel-Excel/blob/f3cce8310b94b4e5716a71da58440ae81c7afe24/composer.json#L24)

So the install is probably blocked by something else.
